### ISSUE TITLE
Little rich girl

### ISSUE THREAD

**Issue by Magneticdud** [Magneticdud/Hitomi-fansub#4]
Yuka is called お嬢さま - this has been translated as "little rich girl". I don't really like this translation.

おじょうさま means "princess", very respectful like "Miss" or "Lady". Sometimes used to indicate a rich/spoiled girl/woman

**Comment by MBjdrma** [comment]
I agree it can have various spins - sometimes ironic, sometimes judgmental, sometimes respectful, sometimes important sign of class status, or of rank...ideally the subs can indicate the intent of the name-appendages that we don't really use in the US and western Europe when speaking with or about people. Kouhai and Sempai are also hard to render...or when someone purposely says "-Sama" instead of "-San." You miss a lot of the subtler nuances and cultural stuff when you're just reading subtitles...
